jsr107/pom.xml

```diff
@@ -1,15 +1,14 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.ehcache.sample</groupId>
-  <artifactId>jsr107</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
 
-  <properties>
-    <jcache.version>1.1.0</jcache.version>
-    <ehcache.version>3.5.2</ehcache.version>
-    <maven-compiler-plugin.version>2.3.1</maven-compiler-plugin.version>
```

Comment:
put it in the parent pom ?

Reply:
Ok. I've moved some more redundancies to the parent